cloverage/project.clj

```diff
@@ -1,3 +1,7 @@
+(def clojure-profile (if (System/getenv "CI")
+                       (-> "CLOJURE_VERSION" System/getenv not-empty (doto (assert "CLOJURE_VERSION is unset!")))
+                       "1.11"))
```

Comment:
huh, that's cool, I have never seen this hack before! (I always used profiles.)

Is there a particular advantage or disadvantage you feel this approach has compared to lein with-profiles?

Reply:
`lein with-profile`  is being used (see usages below) - all what this `def`  does is helping which profile will be selected.

This allows a simple matrix syntax in CI

Reply:
Ohhh, I see, yep, OK, makes sense.